src/modules/site-v2/base/views/strains.py

```diff
@@ -341,21 +341,18 @@ def order_confirmation(invoice_hash):
   if order_obj is None:
     abort(404)
 
-  # Parse the individual items in the order into (1) a dict {name: price} and (2) a list of dicts
-  order_obj["items"] = {
-    x.split(":")[0]: float(x.split(":")[1]) for x in order_obj['items'].split("\n")
-  }
-  items = [ {'strain': k, 'price': v} for k, v in order_obj["items"].items() ]
-
-  # Add species to each strain, if applicable
-  strains = get_strains()
-  for item in items:
-    for strain in strains:
-      if item['strain'] == strain.isotype:
-        item['species'] = strain.species_name
-        break
-      else:
-        item['species'] = ''
+  # Parse the individual items in the order into a list of dicts
+  order_obj["items"] = [ x for x in order_obj['items'].split("\n") ]

```

Comment:
This is a bit pedantic, but since `str.split` outputs a list, you don't need the `[ x for x in ... ]` list comprehension here.

(Sometimes you might see `x for x in ...` if there's an `if` clause after it -- that's how you'd write a filter using list comprehensions.  Alternatively, if all you want to do is convert to a list, you can use the built-in function `list()`.  In this particular case, I think you're safe to use `split` directly.)